### PR TITLE
New ThemeRoller demos

### PR DESCRIPTION
**Pull request by rxaviers**
Reviewing #248 on the stage server, @jzaefferer asked if ThemeRoller demos page could be updated similarly to https://github.com/jquery/download.jqueryui.com/pull/248/files#diff-14 changes. I proposed using the same content for easier maintenance and consistency. This PR implements such.

This change happens to also fix #242.

TODO:
- [ ] Drop temporary changes commit.

### CODE REVIEW

**Comment by rxaviers** [comment]
Some screenshots...

screenshot from 2015-01-27 19 01 47

---

screenshot from 2015-01-27 19 13 09

---

screenshot from 2015-01-27 19 13 17

---

screenshot from 2015-01-27 19 13 23

**Comment by rxaviers** [comment]
Overlay is buggy. I don't know what's going on here.

screenshot from 2015-01-27 19 12 34

**Comment by rxaviers** [comment]
> Overlay is buggy. I don't know what's going on here.

Figured it out. 80b5b18

screenshot from 2015-01-27 22 22 40

**Comment by rxaviers** [comment]
@jzaefferer ready for review

**Comment by jzaefferer** [comment]
* [ ] remove the "(not currently used in UI widgets)" text
* [ ] Update application: Shadow opacity and Corners (also under Dropshadow) should have title "Not in use anymore in 1.12"
* [ ] Use smart position:fixed for application to stay in view

**Comment by rxaviers** [comment]
> remove the "(not currently used in UI widgets)" text

I assume this needs to also be fixed in https://github.com/jquery/download.jqueryui.com/blob/master/template/zip/index-1-12.html#L115, right? What about this same fix for index.html of previous versions (<1.12)?

**Comment by jzaefferer** [comment]
I'm fine with not changing older versions, since they don't use the `ui-widget-shadow` class. We can remove it there as well, since it doesn't really help, and the overlay class is actually used by dialog.

**Comment by rxaviers** [comment]
> Use smart position:fixed for application to stay in view

Became its own ticket https://github.com/jquery/download.jqueryui.com/issues/252.

**Comment by rxaviers** [comment]
> remove the "(not currently used in UI widgets)" text
> Update application: Shadow opacity and Corners (also under Dropshadow) should have title "Not in use anymore in 1.12"

Please, can you verify 27f5f5c

**Comment by jzaefferer** [comment]
Diff looks good.

Didn't you want to reuse the zip template for the TR template, or the other way around? Can refactor that later, as long as its tracked.

**Comment by rxaviers** [comment]
> Didn't you want to reuse the zip template for the TR template, or the other way around? Can refactor that later, as long as its tracked.

Right now, although it's the same content. There are duplicate templates. Ideally, we should use the same template for zip's index.html and themeroller demos #253.

File: template/themeroller/demo.html [hunk 1]
```diff
@@ -0,0 +1,447 @@
+<style>
+body{
+	font-family: "Trebuchet MS", sans-serif;
+	margin: 50px;
+}
```
Comment (rxaviers) [inline template/themeroller/demo.html]: This needs to be fixed.

**Pull request closed without merging**